.github/workflows/integration-tests.yml

```diff
@@ -1,9 +1,6 @@
 name: Integration Tests
 on:
-  pull_request:
-  push:
-    tags:
-      - "*"
+  workflow_dispatch:
```

Comment:
@kalverra why are we switching this to workflow_dispatch?

Reply:
That must have slipped in early in my debug. Thanks for the catch, switched back.